contracts/src/internal/HyperdriveAdmin.sol

```diff
@@ -144,11 +81,11 @@ abstract contract HyperdriveAdmin is IHyperdriveEvents, HyperdriveBase {
     /// @param _target The target token to sweep.
     function _sweep(IERC20 _target) internal nonReentrant {
         // Ensure that the caller is authorized to sweep tokens.
-        address sweepCollector = _sweepCollector;
+        address sweepCollector = _adminController.sweepCollector();
         if (
-            !_pausers[msg.sender] &&
             msg.sender != sweepCollector &&
-            msg.sender != _governance
+            msg.sender != _adminController.hyperdriveGovernance() &&
+            !_isPauser(msg.sender)
```

Comment:
Same with this, should we care who call this as long as the sweepCollector gets the tokens?

Reply:
Similar story to the `collectGovernanceFees` functions.